src/history.md

```diff
@@ -0,0 +1,73 @@
+# History
+
+## BIOS
+
+BIOS is the good old, inscrutable, unstructured, non-standard way of
+initializing a hardware platform in the pre-UEFI days. In other words it's a
+binary blob with no standardized structure, that is responsible for
+initializing CPU and memory, and jumping to a hard-coded position on the MBR of
+the first disk drive.
+
+BIOS has been largely replaced by the (much better) UEFI over the past 20
+years. Many UEFI implementations still offer a "BIOS compatibility mode" which
+make it behave like an old BIOS, with its (lack of) features.
+
+BIOS is also a misused term for system firmware nowadays. You may still hear
```

Comment:
Let's add a link to https://en.wikipedia.org/wiki/BIOS somewhere in the preface. I think it explains the history and situation fairly well.

Reply:
done in #105